vtex-apps/product-quantity
# added props showSuffix

#### What problem is this solving?

We are migrating a cms store to Vtex IO by inserting the component `quantity-selector` was rendered the unit of measure in the quantity selector, becoming different from the proposed layout

#### How to test it?

access the workspace and see the quantity selector showing the unit of measurement
[Workspace showSuffixTrue](https://showsuffix--tokstokio.myvtex.com/triang-caneca-espresso-80-ml-ouro-preto-hipfix/p)

access the workspace and see the quantity selector without the unit of measure
[Workspace showSuffixFalse](https://showsuffixfalse--tokstokio.myvtex.com/triang-caneca-espresso-80-ml-ouro-preto-hipfix/p)


#### Screenshots or example usage:

```diff
 "product-quantity": {
    "props": {
      "showSuffix": true
    }
  },
```
#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

N/A
